bioindex/src/main/resources/compression.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -xe
+
+sudo aws s3 cp s3://dig-data-registry/hail.jar /usr/lib/spark/jars/
```

Comment:
I built the hail.jar from the hail source (it needs to compiled using java 8 since that's what our EMR clusters use) and then uploaded it to S3.  We probably need a better location in s3, but I used this one now since it's not in production use.

Reply:
So for things like this we tend to use `s3://dig-aggregator-data/bin/`